tcsocket/app/worker.py

```diff
@@ -228,9 +230,16 @@ async def delete_old_appointments(ctx):
         logger.info('%d old appointments deleted', v.rowcount)
 
 
+async def kill_worker(ctx):
+    pid = os.getppid()
+    os.kill(os.getppid(), SIGTERM)
+    os.waitpid(pid, 0)
```

Comment:
I think better not to wait, because this will cause the worker to try and wait until this job has finished.